README.md

```diff
@@ -1,5 +1,14 @@
 # Tensor Network Contraction Optimizer (TNCO)
 
+[![python](https://img.shields.io/badge/python-3.8+-blue)]()
+[![c++](https://img.shields.io/badge/c++-17-blue)]()
+<br>
+[![run_tests](https://github.com/google-research/tnco/actions/workflows/run_tests.yml/badge.svg)](https://github.com/google-research/tnco/actions/workflows/run_tests.yml)
+[![cpp_linter](https://github.com/google-research/tnco/actions/workflows/cpp_linter.yml/badge.svg)](https://github.com/google-research/tnco/actions/workflows/cpp_linter.yml)
+[![codeql](https://github.com/google-research/tnco/actions/workflows/github-code-scanning/codeql/badge.svg)](https://github.com/google-research/tnco/actions/workflows/github-code-scanning/codeql)
+<br>
+[![nature](https://img.shields.io/badge/Nature-https%3A%2F%2Fdoi.org%2F10.1038%2Fs41586--025--09526--6-green)](https://doi.org/10.1038/s41586-025-09526-6)
+
 `TNCO` is a heuristic tool that optimizes tensor network contraction paths. It
 represents the contraction as a tree -- with the initial tensors as leaves and
```

Comment:
```suggestion
represents the contraction as a tree – with the initial tensors as leaves and
```

Suggestion: use an en-dash instead of a double hyphen. (Typographically, it used to be the norm to use _em_-bashes without spaces around them, but these days it seems more common to use en-dash and put spaces around them. And yes, in the font used on GitHub, it's almost impossible to tell hyphens apart from em-dashes, but still …)